src/nxdoc/web-ui/web-ui-tutorials/how-to-create-and-customize-nuxeo-actions-menu.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 'HOWTO: Create a Nuxeo actions menu'
+review:
+    comment: ''
+    date: '2020-03-25'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create a Nuxeo actions menu.
+        level: Advanced
+        tool: code
+        topics: Web UI
+labels:
+    - lts2017-ok
+    - lts2019-ok
+    - tutorial
+    - nuxeo-elements
+    - nuxeo-ui-elements
+    - polymer
+tree_item_index: 500
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create and customize a Nuxeo actions menu on your code. The actions menu is a responsive menu that can hold actions (e.g. buttons), which hides behind a dropdown according to the available (or predefined) space.
+{{! /excerpt}}
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/NXDOC/Master/HOWTO: Create a Nuxeo actions menu/nuxeo actions menu screenshot
+    name: nuxeo-action-menu.png
+    1.1.3#screenshot#up_to_date
+--}}
+![nuxeo actions menu screenshot](nx_asset://9c147cdc-311a-4190-b7d1-7d2a255216f2 ?w=300,border=true)
+
+## Create an actions menu:
+First, we start by adding a nuxeo actions menu element.
+
+```
+<nuxeo-actions-menu>
+</nuxeo-actions-menu>
+```
+
+Then we add the menu items, which should consist of actions - such as a nuxeo defined action, for example:
+
+```
+<nuxeo-add-to-collection-button document="[[document]]">
+</nuxeo-add-to-collection-button>
+```
+
+Or you can create your own.
+
+## Create your own action:
+
+First, we import the ***nuxeo-action-button-styles** styles module to make sure the action is styled properly.
+
+```
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-action-button-styles.js';
+```
+
+And include the nuxeo-action-button-styles module in the element style.
+
+```
+<style include="nuxeo-action-button-styles">
+  ...
+</style>
+```
+
+Then, we add a showLabel boolean property (with false as the initial value).
+
+```
+showLabel: {
+    type: Boolean,
+    value: false,
+},
+```
+
+On the html, we add a span element with class label, with your label bound to it, right next to your button, and bind it’s hidden attribute to !showLabel.
+
+```
+<paper-icon-button id="myAction" icon="nuxeo:someicon" on-tap=”_doSomething”>
+</paper-icon-button>
+<span class="label" hidden$="[[!showLabel]]">
+  [[_label]]
+</span>
+```
+
+Finally, we wrap the two elements above (the button and the label) in a new div with class action, and move your tap or click event listener to the div:
```

Comment:
`Finally, we wrap the two elements above (the button and the label) in a new div with class action, and move your tap or click event listener to it`:

Reply:
thanks! suggestion added